.gitignore

```diff
@@ -58,3 +58,4 @@ test_app
 vendor/bundle
 */vendor/bundle
 .ignore
+TODO
```

Comment:
What's the purpose of this file? If this is your personal file it would a good idea to add the entry to global .gitignore: https://robots.thoughtbot.com/global-gitignore